device/dell/x86_64-dellemc_z9264f_c3538-r0/plugins/psuutil.py

```diff
@@ -28,15 +30,22 @@ class PsuUtil(PsuBase):
 
     def __init__(self):
         PsuBase.__init__(self)
+     
+    def isDockerEnv(self):
```

Comment:
It might be a bit clearer if this function returned true/false based on num_docker > 0.

Reply:
Hi Joe,
 
    Addressed your valuable comments...
Can you please review.

Thanks & Regards,
Karthik.G